src/session/session_prefetch.c

```diff
@@ -15,6 +15,10 @@
 bool
 __wt_session_prefetch_check(WT_SESSION_IMPL *session, WT_REF *ref)
 {
+    /* Disable pre-fetch work on tiered tables. */
```

Comment:
There's also a `"tier:"` form of the URI - I'm not totally sure about whether that form is still a normal btree. But besides that, I think it would be more robust to check the type (i.e. `dhandle->type == WT_DHANDLE_TYPE_TIERED`, and maybe also check for `WT_DHANDLE_TYPE_TIERED_TREE` if that ends up necessary).

Reply:
I've changed this to check `dhandle->type` for both `WT_DHANDLE_TYPE_TIERED` and `WT_DHANDLE_TYPE_TIERED_TREE`. As far as I can see, both types refer to tiered tables

Reply:
Cool - thanks.